Dfe.Academies.External.Web/Pages/WhatYouWillNeed.cshtml

```diff
@@ -5,6 +5,12 @@
 }
 
 <div class="govuk-grid-column-two-thirds">
+
+    <a href="Index" class="govuk-back-link">Back to start your application</a>
```

Comment:
hi @mokoruwaDfE , the back link is part of the layout page. So this HTML can be replaced by this:-
@section BeforeMain
{
    <a asp-page="YourApplications" class="govuk-back-link">Back</a>
}

The same as ApplicationOverview page